test/unit/collection-view.spec.js

```diff
@@ -467,6 +467,39 @@ describe('collection view', function() {
     });
   });
 
+  describe('when a number of models are added to a non-empty collection with an "at" option', function() {
+    beforeEach(function() {
+      this.model1 = new Backbone.Model({foo: '1'});
+      this.model2 = new Backbone.Model({foo: '2'});
+      this.collection = new Backbone.Collection([this.model1, this.model2]);
+
+      this.collectionView = new this.CollectionView({
+        collection: this.collection
+      });
+      this.collectionView.render();
+
+      this.model3 = new Backbone.Model({foo: '3'});
+      this.model4 = new Backbone.Model({foo: '4'});
+      this.collection.add([this.model3, this.model4], {at: 1});
+
+      this.model5 = new Backbone.Model({foo: '5'});
+      this.model6 = new Backbone.Model({foo: '6'});
+      this.collection.add([this.model5, this.model6], {at: 2});
+    });
+
+    it('should add models and render views in right order', function() {
+      var order = _.pluck(this.collectionView.$el.find('span'), 'innerHTML');
+      var backboneVersion = _.map(Backbone.VERSION.split('.'), function(x) {
+        return parseInt(x);
+      });
+      if (backboneVersion[0] >= 1 && backboneVersion[1] >= 2) {
+        expect(order).to.deep.equal(['1', '3', '5', '6', '4', '2']);
```

Comment:
can we make these numbers mean something. My hope is that current backbone should have sequential numbers 1..6 and maybe older backbone has busted numbers... now they're both funky